Classes/WPEditorView.m

```diff
@@ -368,11 +368,63 @@ - (void)stopObservingKeyboardNotifications
 
 - (void)keyboardDidShow:(NSNotification *)notification
 {
+    BOOL isiOS7OrEarlier = ([[[UIDevice currentDevice] systemVersion] floatValue] < 8.0);
+    
+    if (isiOS7OrEarlier) {
+        // PROBLEM: under iOS 7, it seems that setting the proper insets in keyboardWillShow: is not
+        // enough.  We were having trouble when adding images, where the keyboard would show but the
+        // insets would be reset to {0, 0, 0, 0} between keyboardWillShow: and keyboardDidShow:
```

Comment:
:weary: nice job with finding this.  I swear we need a blog post that enumerates all of the workarounds we have done!